custom/src/main/java/com/splunk/opentelemetry/micrometer/MicrometerInstaller.java

```diff
@@ -44,6 +41,10 @@ public void beforeAgent(Config config) {
     }
   }
 
+  public int order() {
+    return 1;
```

Comment:
Can you add a comment explaining the value? E.g. `needs to run after OpenTelemetryInstaller`

Reply:
@mateuszrzeszutek what do you think if I changed `beforeAgent` -> `afterAgent` then `order` wouldn't be needed?

Reply:
I think it's safer to initialize metrics before any instrumentation is applied and tries to use it, I'd probably prefer using `beforeAgent()` and `order()` to be extra safe and avoid any potential timing/race issues.
I guess we can just leave a TODO mentioning https://github.com/open-telemetry/opentelemetry-java/issues/3924 somewhere in the code so that it doesn't get lost.

Reply:
Added comment to `order()`. I believe that instead of adding a TODO we can rely on @breedx-splk to keep track of that issue.